packages/paste-website/src/pages/components/help-text/index.mdx

```diff
@@ -0,0 +1,342 @@
+---
+title: Help Text - Components
+package: '@twilio-paste/help-text'
+description: 'Help text that can be paired with a Paste form component'
+slug: /components/help-text/
+---
+
+import {graphql} from 'gatsby';
+import {Label} from '@twilio-paste/label';
+import {Input} from '@twilio-paste/input';
+import {Select, Option} from '@twilio-paste/select';
+import {TextArea} from '@twilio-paste/textarea';
+import {HelpText} from '@twilio-paste/help-text';
+import {Anchor} from '@twilio-paste/anchor';
+import {Box} from '@twilio-paste/box';
+import {Text} from '@twilio-paste/text';
+import {Button} from '@twilio-paste/button';
+import {Table, THead, TBody, Td, Th, Tr} from '@twilio-paste/table';
+import {InformationIcon} from '@twilio-paste/icons/esm/InformationIcon';
+import {UnorderedList, ListItem} from '@twilio-paste/list';
+import {Callout, CalloutTitle, CalloutText} from '../../../components/callout';
+import {DoDont, Do, Dont} from '../../../components/DoDont';
+import {Codeblock} from '../../../components/codeblock';
+import {SidebarCategoryRoutes} from '../../../constants';
+import Changelog from '@twilio-paste/help-text/CHANGELOG.md';
+
+export const pageQuery = graphql`
+  {
+    allPasteComponent(filter: {name: {eq: "@twilio-paste/help-text"}}) {
+      edges {
+        node {
+          name
+          description
+          status
+          version
+        }
+      }
+    }
+    mdx(fields: {slug: {eq: "/components/help-text/"}}) {
+      fileAbsolutePath
+      frontmatter {
+        slug
+        title
+      }
+      headings {
+        depth
+        value
+      }
+    }
+    allAirtable(filter: {data: {Feature: {eq: "HelpText"}}}) {
```

Comment:
```suggestion
    allAirtable(filter: {data: {Feature: {eq: "Help Text"}}}) {
```

Adding that space will get the correct Airtable info, and populate the status header.